static/js/marker.js

```diff
@@ -148,7 +148,7 @@ var MarkerView = Backbone.View.extend({
                 break;
             case 'multiple':
                 accuracy = (this.marker.opacity != 1);
-                markerTitle = 'מספר תאונות בנקודה זו';
+                markerTitle = 'תאונות בנקודה זו';
```

Comment:
At first I didn't understand why you did that. Care to add a comment that the number will be added by the code?